singleuser/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # Build as renku/singleuser
 # Run with the DockerSpawner in JupyterHub
 
-ARG JUPYTERHUB_VERSION=0.9
+ARG JUPYTERHUB_VERSION=0.9.2
```

Comment:
they made `0.9.1` disappear, will this one last?

Reply:
it's there: https://github.com/jupyterhub/jupyterhub/releases/tag/0.9.1